static/app/components/nav/sidebar.tsx

```diff
@@ -140,14 +138,21 @@ const SidebarItemWrapper = styled('li')`
     line-height: 177.75%;
     border: 1px solid transparent;
 
-    &:hover {
+    &:hover,
+    &:focus-visible {
       color: var(--color-hover, ${theme.white});
+      background: rgba(255, 255, 255, 0.1);
```

Comment:
Let's use the `InteractionStateLayer` component for the hover/press effect. It helps standardize the background color and works in both light and dark mode.

Reply:
Ah yep, missed that! Great callout.

Reply:
Addressed in [285de0b](https://github.com/getsentry/sentry/pull/79223/commits/285de0b4d2f4857492f9e054ee9adc0f3a1b00fd)